src/timeline/TimelineModel.h

```diff
@@ -126,6 +126,8 @@ class TimelineModel : public QAbstractListModel
           int currentIndex READ currentIndex WRITE setCurrentIndex NOTIFY currentIndexChanged)
         Q_PROPERTY(std::vector<QString> typingUsers READ typingUsers WRITE updateTypingUsers NOTIFY
                      typingUsersChanged)
+        Q_PROPERTY(QString reaction READ reaction WRITE setReaction NOTIFY reactionChanged RESET
```

Comment:
I'm pretty confident that none of these reaction additions are needed.  Should be able to remove this, the reactAction, reaction, setReaction, resetReaction, and reactionChanged functions.  I'll test when finished to make sure I don't break anything.

Reply:
Yeah, that even looks counterintuitive.

Reply:
I think when I first started implementing this, I envisioned it being similar to the replies.  But then I changed my approach and didn't clean up.